src/components/App/App.component.jsx

```diff
@@ -1,57 +1,37 @@
-import React, { useLayoutEffect } from 'react';
+import React, { useReducer } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
-import AuthProvider from '../../providers/Auth';
 import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
-
-function App() {
-  useLayoutEffect(() => {
-    const { body } = document;
+import VideoDetail from '../../pages/VideoDetail/VideoDetail.page';
```

Comment:
Ya lo comenté en el anterior PR pero para reforzar, podemos reducir esta línea a lo siguiente gracias a que ya tenemos un index.js
```suggestion
import VideoDetail from '../../pages/VideoDetail';
```